docs/how-to-guides/curating-the-editor-experience/theme-json.md

```diff
@@ -0,0 +1,210 @@
+# theme.json
```

Comment:
Find the lower case t to throw me off here. Any reason not to capitalize?

Reply:
I haven't tested across different systems, but I'd always err on the side of preserving the correct case for filenames.

Reply:
> I haven't tested across different systems, but I'd always err on the side of preserving the correct case for filenames.

Yeah, we are doing the same thing with other common filenames like block.json, functions.php, etc. That said, we do need to comb through the Handbook and standardize this.